nucypher/cli/commands/stake.py

```diff
@@ -1399,3 +1400,35 @@ def migrate(general_config: GroupGeneralConfig,
                           emitter=emitter,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='migrate')
+
+    paint_staking_accounts(emitter=emitter,
+                           signer=STAKEHOLDER.signer,
+                           registry=STAKEHOLDER.registry,
+                           domain=STAKEHOLDER.domain)
+
+
+@stake.group()
+def rewards():
+    """Manage staking rewards."""
+
+
+@rewards.command('show')
+@group_staker_options
+@option_config_file
+@group_general_config
+@click.option('--num_past_periods', help="Number of past periods for which to calculate rewards", type=click.INT)
```

Comment:
Use hyphens instead of underscores for CLI options please 😄 

```suggestion
@click.option('--num-past-periods', help="Number of past periods for which to calculate rewards", type=click.INT)
 ```
 
 Also - what do you think about shortening this option to just `--periods`?  I tend to prefer shorter CLI flags.
 
 ```suggestion
@click.option('--periods', help="Number of past periods for which to calculate rewards", type=click.INT)
 ```